src/tests/ftest/soak/smoke.py

```diff
@@ -24,7 +24,7 @@ def test_soak_smoke(self):
         various jobs defined in the soak yaml.  It will run for no more than
         20 min
 
-        :avocado: tags=manual
+        :avocado: tags=all,full_regression
```

Comment:
Is this meant to run hw,large or hw,large,24 in full_regression?

Reply:
`hw,large,24` is intentional to support running in HW Large and a new stage in the `soak_testing` branch